static/script.js

```diff
@@ -0,0 +1,85 @@
+    $(function(){
+    var answer = null;
+    $("body").on("click", "#show_prediction", function(event){
+        $("#prediction").toggle();
+    });
+
+    /*---- SITE WIDE AJAX ----*/
+    $(".navlink").on("click", function(event){
+        event.preventDefault();
+        var ref = $(this).attr("href")
+
+        $.ajax({
+            method : "GET",
+            url : ref,
+        }).done(function(response){
+            $("main").replaceWith($($.parseHTML(response)).filter("main"))
+            /*alert($("main").html())*/
+        }).fail(function(){
+            alert("Something went wrong")
+        });
+    });
+
+    /*---- QUESTION PAGE AJAX ----*/
+    $("body").on("click", "input:radio[name=answer]", function(event){
+        answer = $(this).val();
+    });
+
+    $("body").on("submit", '.question_form', function(event){
+        event.preventDefault();
+        $("#submit").attr('disabled', true);
+        var question_id = $("#qu").val();
+        var score = $("#score").val();
+        var predict = $("#predict").val();
+        var count = $("#count").val();
```

Comment:
if, rather than extracting the value from these elements right away you were instead to set variables to the elements themselves, then you could use the same variables both to extract the values and to set new values on `done` below.  It would save you traversing the DOM twice to find the same element over again.